src/components/ATATExpandableLink.vue

```diff
@@ -31,5 +39,7 @@ export default class ExpandableLink extends Vue {
   private open = false;
```

Comment:
AC 1.b has default state expanded, and AC 1.e.ii requires accordion to remain in the state it was in when user leaves the page and comes back. So we'll need to change `open` to a `PropSync` value with default `false`, and also store the state in Description of Work store `src/store/descriptionOfWork/index.ts` in a var like `isDOWSummaryAccordionOpen` or something to that effect, and toggle that value when user clicks. Let me know if you need more info.

Reply:
Okay, I think I can figure this out but I'll ping you if I get blocked!